be/src/vec/aggregate_functions/aggregate_function_window_funnel.h

```diff
@@ -212,64 +315,97 @@ struct WindowFunnelState {
         } else {
             window_funnel_mode = WindowFunnelMode::DEFAULT;
         }
-        sorted = true;
     }
 
     void write(BufferWritable& out) const {
-        write_var_int(max_event_level, out);
+        write_var_int(event_count, out);
         write_var_int(window, out);
         if (enable_mode) {
             write_var_int(static_cast<std::underlying_type_t<WindowFunnelMode>>(window_funnel_mode),
                           out);
         }
-        write_var_int(events.size(), out);
-
-        for (int64_t i = 0; i < events.size(); i++) {
-            int64_t timestamp = binary_cast<DateValueType, NativeType>(events[i].first);
-            int event_idx = events[i].second;
-            write_var_int(timestamp, out);
-            write_var_int(event_idx, out);
+        PBlock pblock;
```

Comment:
Will there be compatibility issues here?

Reply:
It's inevitable, the previous data structure cannot express the semantic of window_funnel.